store/modules/heatmap/index.js

```diff
@@ -4,16 +4,29 @@ import getters from "./getters";
 import mutations from "./mutations";
 
 const default_state = {
-  // heatmap_values: 24 element array. each element corresponds to a well in the plate
-  heatmap_values: [],
+  heatmap_values: {
+    "Twitch Force": { data: [], range_min: 0, range_max: 100 },
+    "Twitch Period": { data: [], range_min: 0, range_max: 100 },
+    "Twitch Frequency": { data: [], range_min: 0, range_max: 100 },
+    "Twitch Width 80": { data: [], range_min: 0, range_max: 100 },
+    "Contraction Velocity": { data: [], range_min: 0, range_max: 100 },
+    "Relaxation Velocity": { data: [], range_min: 0, range_max: 100 },
+  },
   heatmap_idx: null,
-  heatmap_display_array: [],
-  heatmap_display_idx: null,
+  heatmap_display_array: [
+    "Twitch Force",
+    "Twitch Period",
+    "Twitch Frequency",
+    "Twitch Width 80",
+    "Contraction Velocity",
+    "Relaxation Velocity",
+  ],
+  heatmap_display_idx: 0,
   heatmap_display_min_max: [],
   heatmap_display_user_min_max: {},
   heatmap_autoscale: false,
-  heatmap_options_array: [],
-  heatmap_options_idx: null,
+  heatmap_options_array: ["Warm", "Cool", "Blue/Red", "Purple/Green"],
```

Comment:
delete this, the options get set from the gradient store values now

Reply:
Got it!